src/dakara_feeder/dakara_feeder.py

```diff
@@ -48,16 +49,19 @@ def __init__(self, config, force_update=False, progress=True):
         self.songs_per_chunk = config["server"].get("songs_per_chunk", SONGS_PER_CHUNK)
         self.bar = progress_bar if progress else null_bar
         self.song_class_module_name = config.get("custom_song_class")
-        self.Song = None
+        self.song_class = BaseSong
```

Comment:
Oh, nice, I like it better this way, default value set, and changed in case of override